src/schema/v2/__tests__/sale_artwork.test.js

```diff
@@ -665,6 +685,18 @@ describe("SaleArtwork type", () => {
       })
     })
 
+    it("returns 'Ends date/time' when the sale has started, end_at has passed but bidding was extended", async () => {
+      saleArtwork.ended_at = null
+      saleArtwork.end_at = "2029-02-17T11:00:00+00:00"
```

Comment:
Yes this is bad 😄 

All the specs in this block are of this form and use the 2029 year, and I wanted to focus on getting the websocket stuff working so didn't want to update all these specs. We can come back and update the block to stub current time (vs. having these specs fail for 7-years-from-now us)!